### PR TITLE
html-attribute-id-addmedia

### PR DESCRIPTION
**Pull request by ROOHITH**
### Type of Change

- Editing an existing entry (fixing a typo, bug, issues, etc)

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Change the image description that you are using in `[]`. Make them a bit descriptive about the image. As if the images/gif failed to load then this text will show there instead.

**Comment by ROOHITH** [comment]
I have made the changes that were requested.

**Review by dakshdeepHERE — APPROVED** [review]
@ROOHITH looks good for second 2nd review

File: content/html/concepts/attributes/terms/id/id.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ In the example below, the `<p>` (paragraph) element is given a unique identifier
 </p>
 ```
 
+![Screenshot that shows an example of a unique id attribute.](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example-img1.png)
```
Comment (ishg-153) [inline content/html/concepts/attributes/terms/id/id.md:48]:
```suggestion
![Use of unique id attribute with inline CSS](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example-img1.png)
```

File: content/html/concepts/attributes/terms/id/id.md [hunk 2]
```diff
@@ -71,6 +73,8 @@ A link can also target a location on another page:
 
 Clicking on the link in the examples above will direct the browser screen to the location where `id` is `"my-city"` (either on the same page or on another page).
 
+![Shows how navigating a section on the same page and another page uses a unique id attribute.](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example2-gif.gif)
```
Comment (ishg-153) [inline content/html/concepts/attributes/terms/id/id.md:76]:
```suggestion
![Use of a unique id attribute for navigation on the same and other pages in the website](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example2-gif.gif)
```

File: content/html/concepts/attributes/terms/id/id.md [hunk 3]
```diff
@@ -87,6 +91,8 @@ Styling via `id` in a stylesheet:
 
 The HTML element with `id` is selected in CSS for styling using `#` followed by the `id` value. This applies to internal or external ruleset styling methods. The example above applies a color to the designated element, for an overview of all styling methods, please see [CSS Anatomy](https://www.codecademy.com/resources/docs/css/anatomy).
 
+![CSS styling using id selector.](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-exampl3.png)
```
Comment (ishg-153) [inline content/html/concepts/attributes/terms/id/id.md:94]:
```suggestion
![CSS styling using id selector](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-exampl3.png)
```

File: content/html/concepts/attributes/terms/id/id.md [hunk 4]
```diff
@@ -102,3 +108,5 @@ function changeColor(newColor) {
 ```
 
 In the example above, the element is selected using the `getElementById()` method. The color of that element is then changed by assigning a new value, that is passed in as an argument of the function, to the `style.color` property.
+
+![The javascript uses the id attribute to style the html element.](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example4.png)
```
Comment (ishg-153) [inline content/html/concepts/attributes/terms/id/id.md:112]:
```suggestion
![Use of id attributes to style HTML elements in JavaScript](https://raw.githubusercontent.com/Codecademy/docs/main/media/html-attribute-id-example4.png)
```

**Review by ishg-153 — CHANGES_REQUESTED** [review]
@ROOHITH I have suggested changes in the `alt-text`. Please make those.

**Review by ishg-153 — APPROVED** [review]
@ROOHITH Great! This is ready to merge.

**Pull request merged**